src/components/Layout/Header/GlobalSearch/GlobalSearchModal.tsx

```diff
@@ -8,6 +8,7 @@ import {
   useEventListener,
   useUpdateEffect,
 } from '@chakra-ui/react'
+import * as Sentry from '@sentry/react'
```

Comment:
prefer named imports where possible

```suggestion
import { setContext, captureException } from '@sentry/react'
```

Reply:
da518cd68